tests/e2e/scenarios/test_chat.py

```
@@ -1,76 +1,196 @@
-    last_user = user_msgs.last
-    user_text = await last_user.text_content()
-    assert "2+2" in user_text or "2 + 2" in user_text
+async def _send(page, text: str):
+    composer = page.get_by_label("Message composer")  # aria-label, en-GB default
+    await composer.wait_for(state="visible", timeout=5000)
+    await composer.fill(text)
+    send = page.locator(SEL["chat_send"])
+    await send.click()
+
+
+async def _assistant_markdown_contains(page, needle: str, *, timeout: int = 60000):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_assistant_markdown_contains`

Add return type annotation: `None`

(ANN202)